frontend/src/views/Bridge.vue

```diff
@@ -347,13 +383,15 @@ export default Vue.extend({
 
     //check current net by checking url
     const env = window.location.href.startsWith('https://test') ? 'test' : 'production'; //const env = 'test';
-    console.log('currentNet: ', env);
     const conf = config as any;
     for(let i = 0; i < this.supportedChains.length; i++){
       this.supportedChainIds.push(conf.environments[env].chains[this.supportedChains[i]].VUE_APP_NETWORK_ID);
     }
   },
-
+  async mounted(){
+    if (!this.contracts.NFTStorage) return;
+    this.bridgeFee = await this.contracts.NFTStorage.methods.getBridgeFee().call({ from: this.defaultAccount });
```

Comment:
.call(defaultCallOptions(state));

Reply:
Also, please extract this to store.ts, we shouldn't have contract calls in components

Reply:
As discussed, we'll leave it here, as we don't have a better solution on hand.